### PR TITLE
feat/RENTAL_REBASE_odometer_image

### PR DESCRIPTION
**Pull request by max-keviv**
- add the odometer image entry in the media file

### CODE REVIEW

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs [hunk 1]
```diff
@@ -390,54 +381,46 @@ otpRideCreate driver otpCode booking = do
...
   pure $ OdometerReadingRes image
+  where
+    startOdometerImagePath :: DRide.Ride -> Flow Text
+    startOdometerImagePath ride = do
+      let fileName = T.pack "startOdometerReadingImage"
+      pathPrefix <- asks (.s3Env.pathPrefix)
+      merchantId' <- pure ride.merchantId >>= fromMaybeM (InvalidRequest "merchantId not present")
+      return
+        ( pathPrefix <> "/ride-odometer-reading/" <> "org-" <> (merchantId'.getId) <> "/"
+            <> (ride.driverId.getId)
```
Comment (0utkarsh) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs:406]: Why `driverId`?

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Ride.hs [hunk 2]
```diff
@@ -88,26 +88,20 @@ type API =
                     :> "cancel"
                     :> ReqBody '[JSON] CancelRideReq
                     :> Post '[JSON] RideCancel.CancelRideResp
+                    :<|> TokenAuth
+                    :> Capture "rideId" (Id Ride.Ride)
+                    :> "odometerReading"
+                    :> QueryParam "start" Bool
```
Comment (0utkarsh) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Ride.hs:94]: Why this param?

**Review by 0utkarsh — CHANGES_REQUESTED** [review]

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Ride.hs (hunk 2 above)
Comment (max-keviv) [reply Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Ride.hs:94]: it is used to identify whether we want to get start image or end image ?

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs (hunk 1 above)
Comment (max-keviv) [reply Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs:406]: using it to make path

**Review by 0utkarsh — CHANGES_REQUESTED** [review]

**Pull request merged**